crypto3/libs/algebra/CMakeLists.txt

```diff
@@ -36,7 +36,9 @@ target_link_libraries(${CMAKE_WORKSPACE_NAME}_${CURRENT_PROJECT_NAME} INTERFACE
         ${CMAKE_WORKSPACE_NAME}::multiprecision)
 
 include(CMTest)
-cm_add_test_subdirectory(test)
+if(BUILD_CRYPTO3_TESTS)
+    cm_add_test_subdirectory(test)
+endif()
```

Comment:
As I remember, it was done like this before, we removed redundant `if` because it is already handled inside `cm_add_test_subdirectory` command. Why do we need to reintroduce it here?